org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -2789,8 +2789,8 @@ public void zoom()
 	        fireValueChange(event);
     	}
     	else
-    	{    		
-    		int AD_Table_ID = MTable.getTable_ID(p_tableName);
+    	{
+    		int AD_Table_ID = infoWindow.getAD_Table_ID();
```

Comment:
hi @CarlosRuiz-globalqss , zoom() also call from InfoGeneralPanel and infoWindow can be null here.

Reply:
Hi @hengsin - I think the user cannot zoom from an InfoGeneralPanel, but anyways I added a check for null just in case.